networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc

```diff
@@ -10,19 +10,46 @@ As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin
 
 To learn more about OVN-Kubernetes, read xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes#about-ovn-kubernetes[About the OVN-Kubernetes network plugin].
 
+// Offline migration to the OVN-Kubernetes network plugin overview
 include::modules/nw-ovn-kubernetes-migration-about.adoc[leveloffset=+1]
+
+// How the offline migration process works
 include::modules/nw-network-plugin-migration-process.adoc[leveloffset=+2]
+
+// Migrating to the OVN-Kubernetes network plugin by using the offline migration method
 include::modules/nw-ovn-kubernetes-migration.adoc[leveloffset=+2]
 
+// Limited live migration to the OVN-Kubernetes network plugin overview
 include::modules/nw-ovn-kubernetes-live-migration-about.adoc[leveloffset=+1]
+
+// How the live migration process works
 include::modules/how-the-live-migration-process-works.adoc[leveloffset=+2]
+
+// Migrating to the OVN-Kubernetes network plugin by using the limited live migration method
 include::modules/nw-ovn-kubernetes-live-migration.adoc[leveloffset=+2]
+
+// Checking cluster resources before initiating the limited live migration
 include::modules/checking-cluster-resources-before-initiating-limited-live-migration.adoc[leveloffset=+3]
+
+// Removing egress router pods before initiating the limited live migration
 include::modules/removing-egress-router-pods-before-initiating-limited-live-migration.adoc[leveloffset=+3]
+
+// Removing the NodeNetworkConfigurationPolicy (NNCP) custom resource (CR)
+include::modules/removing-nncp-limited-live-migration.adoc[leveloffset=+3]
+
+// Initiating the limited live migration process
 include::modules/initiating-limited-live-migration.adoc[leveloffset=+3]
+
+// Patching OVN-Kubernetes address ranges
 include::modules/patching-ovnk-address-ranges.adoc[leveloffset=+3]
+
+// Checking cluster resources after initiating the limited live migration
```

Comment:
Thank you for cleaning up our notes on this fairly complicated migration procedure. Well done!